SU-SWS/stanford_profile
# D8CORE-476 Default layout set for home page

# READY FOR REVIEW

# Summary
- Use layout library module to set default content home page to be full width

# Need Review By (Date)
- 9/26

# Urgency
- Medium

# Steps to Test
1. in cardinal sites `composer require su-sws/stanford_profile:dev-D8CORE-476-default-homepage-layout`
1. `blt drupal:install`
1. verify the home page banner is full width and there is not a sidebar menu.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
